gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/users/UsersResource.java

```diff
@@ -138,6 +138,36 @@ public void create(
                 .subscribe(response::resume, response::resume);
     }
 
+    @POST
+    @Path("/bulk")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(
+            operationId = "bulkCreateOrganisationUser",
+            summary = "Create a platform user or Service Account",
+            description = "User must have the ORGANIZATION_USER[READ] permission on the specified organization")
+
+    @ApiResponse(responseCode = "201", description = "Users or Service Accounts successfully created",
+            content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = NewOrganizationUser.class)))
+    @ApiResponse(responseCode = "500", description = "Internal server error")
+    public void create(
+            @PathParam("organizationId") String organizationId,
+            @Valid @NotNull final BulkRequest<NewOrganizationUser> newOrganizationUsers,
```

Comment:
question: should we really use a NewUser ? I have some doubt as IMO the update user using bulk should also be done on this bulk endpoint using a POST method.

Reply:
(see answer in domain's UsersResource)